templates/zerver/help/analytics.md

```diff
@@ -0,0 +1,85 @@
+# Analytics
+
+Zulip provides [analytics](/stats) for tracking various statistics about your
+organization to create a holistic view of its communications. The graphs are all
```

Comment:
Maybe "All graphs are fully updated once a day, except the **Messages sent over time** graph which is updated once an hour."?